code/API_definitions/sim-swap-notification-subscription.yaml

```
@@ -528,13 +536,24 @@ components:
         phoneNumber:
           $ref: '#/components/schemas/PhoneNumber'
         terminationReason:
-          type: string
-          enum: ["SUBSCRIPTION_EXPIRED", "NETWORK_TERMINATED"]
+          $ref: "#/components/schemas/TerminationReason"
         subscriptionId:
           $ref: "#/components/schemas/SubscriptionId"
         terminationDescription:
           type: string
 
+    TerminationReason:
+      type: string
+      description: |
+        NETWORK_TERMINATED - API server stopped sending notification
+        SUBSCRIPTION_EXPIRED - Subscription expire time (optionally set by the requester) has been reached
+        MAX_EVENTS_REACHED - maximum number of events (optionally set by the requester) has been reached

```

Just a suggestion so it shows properly in the ui

```suggestion
        - NETWORK_TERMINATED - API server stopped sending notification
        - SUBSCRIPTION_EXPIRED - Subscription expire time (optionally set by the requester) has been reached
        - MAX_EVENTS_REACHED - Maximum number of events (optionally set by the requester) has been reached

```